src/test/java/org/junitpioneer/jupiter/CartesianProductTestExtensionTests.java

```diff
@@ -29,7 +29,6 @@
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
 import org.junit.platform.commons.PreconditionViolationException;
-import org.junitpioneer.jupiter.CartesianEnumSource.Mode;
```

Comment:
Have removed this import to avoid a (non-suppressable) deprecation warning for it. However, that has the disadvantage that `Mode` usage in the tests below is a bit verbose.

Should I revert this?